src/test/PoolTestBase.sol

```diff
@@ -17,25 +18,25 @@ abstract contract PoolTestBase is ILockCallback {
     }
 
     function _take(Currency currency, address recipient, int128 amount, bool withdrawTokens) internal {
-        assert(amount < 0);
+        assertGt(amount, 0);
         if (withdrawTokens) {
-            manager.take(currency, recipient, uint128(-amount));
+            manager.take(currency, recipient, uint128(amount));
         } else {
-            manager.mint(recipient, currency.toId(), uint128(-amount));
+            manager.mint(recipient, currency.toId(), uint128(amount));
         }
     }
 
     function _settle(Currency currency, address payer, int128 amount, bool settleUsingTransfer) internal {
-        assert(amount > 0);
+        assertLt(amount, 0);
```

Comment:
same here why not just have this be a positive amount?

it will always be negative passed in but then its always flipped again below. i think itd be much easier to read if it was just always positive